sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/OpenAIClient.java

```diff
@@ -687,4 +703,356 @@ Response<BinaryData> getChatCompletionsWithAzureExtensionsWithResponse(
         return this.serviceClient.getChatCompletionsWithAzureExtensionsWithResponse(
                 deploymentOrModelName, chatCompletionsOptions, requestOptions);
     }
+
+    public String getAudioTranslationText(
+            String deploymentOrModelName, AudioTranslationOptions audioTranslationOptions, String fileName) {
+        List<AudioTranscriptionFormat> acceptedFormats = new ArrayList<>();
+        acceptedFormats.add(AudioTranscriptionFormat.TEXT);
+        acceptedFormats.add(AudioTranscriptionFormat.VTT);
+        acceptedFormats.add(AudioTranscriptionFormat.SRT);
+        if(!acceptedFormats.contains(audioTranslationOptions.getResponseFormat())) {
+            throw new IllegalArgumentException("This operation does not support the requested audio format");
+        }
+        MultipartDataHelper helper = new MultipartDataHelper();
+        MultipartDataSerializationResult result = helper.serializeRequest(audioTranslationOptions, fileName);
+        return getAudioTranslation(deploymentOrModelName, result, helper.getBoundary()).toString();
+    }
+
+    public AudioTranscription getAudioTranslation(
+            String deploymentOrModelName, AudioTranslationOptions audioTranslationOptions, String fileName) {
+        List<AudioTranscriptionFormat> acceptedFormats = new ArrayList<>();
+        acceptedFormats.add(AudioTranscriptionFormat.JSON);
+        acceptedFormats.add(AudioTranscriptionFormat.VERBOSE_JSON);
+        if(!acceptedFormats.contains(audioTranslationOptions.getResponseFormat())) {
+            throw new IllegalArgumentException("This operation does not support the requested audio format");
```

Comment:
We should throw through `ClientLogger`, see [Azure SDK for Java API guideline - logger](https://azure.github.io/azure-sdk/java_implementation.html#logging)

```
throw logger.logThrowableAsError(new IllegalArgumentException("This operation does not support the requested audio format"));
```

Reply:
I don't have an instance of `logger` in this class. Is this available as a static method? Or are you suggesting we add a member for logger?

Reply:
We can create a ClientLogger var `logger` as our class member. Such as 
```
private final ClientLogger logger = new ClientLogger(OpenAIClient.class);
```
and use `logger` to throw error/warning/etc.

```
throw logger.logThrowableAsError(new IllegalArgumentException("This operation does not support the requested audio format"));
```